src/StripeTests/Services/OAuth/OAuthTokenServiceTest.cs

```diff
@@ -0,0 +1,89 @@
+namespace StripeTests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    using Stripe;
+    using Xunit;
+
+    public class OAuthTokenServiceTest : BaseStripeTest
+    {
+        private const string OrderId = "or_123";
+
+        private readonly OAuthTokenService service;
+        private readonly OAuthTokenCreateOptions createOptions;
+        private readonly OAuthTokenDeauthorizeOptions deauthorizeOptions;
+
+        public OAuthTokenServiceTest(MockHttpClientFixture mockHttpClientFixture)
+            : base(mockHttpClientFixture)
+        {
+            this.service = new OAuthTokenService();
+
+            this.createOptions = new OAuthTokenCreateOptions
+            {
+                Code = "this_is_an_authorization_code",
+                GrantType = "authorization_code",
+            };
+
+            this.deauthorizeOptions = new OAuthTokenDeauthorizeOptions
+            {
+                StripeUserId = "acct_test_deauth",
+            };
+        }
+
+        [Fact]
+        public void Create()
+        {
+            // stripe=mock doesn't support OAuth endpoints, so stub the request
```

Comment:
Not the right place but: should we add support for it?

Reply:
Probably? It's arguably a part of the API. I don't see it happening soon though, the backend implementation is completely independent from the rest of the API and is not ready at all to be encoded into the OpenAPI spec.